torchci/pages/nightlies.tsx

```diff
@@ -280,31 +278,6 @@ export default function Page() {
                 value: "audio",
               },
             ]}
-            queryCollection="nightlies"
-            columns={[
-              { field: "COUNT", headerName: "Count", flex: 1 },
-              { field: "name", headerName: "Name", flex: 4 },
-            ]}
-            dataGridProps={{ getRowId: (el: any) => el.name }}
-          />
-        </Grid>
-
-        <Grid item xs={6} height={ROW_HEIGHT}>
-          <NightlyJobsRedPanel params={timeParams} repo={"text"} />
```

Comment:
Did you mean to get rid of this?

Reply:
Yes text repo is no longer in development